crypto/evp_extra/p_pqdsa_test.cc

```diff
@@ -1577,6 +1596,107 @@ TEST_P(PQDSAParameterTest, MarshalParse) {
             Bytes(pkey->pkey.pqdsa_key->public_key, GetParam().public_key_len));
 }
 
+// Helper function that:
+// 1. Creates a memory BIO
+// 2. Writes the provided DER data to BIO in PEM format
+// 3. Determines resulting PEM size and allocates buffer
+// 4. Reads PEM data into output buffer
+// 5. Returns the PEM string and length via out parameters
+static bool DER_to_PEM(const uint8_t* der, size_t der_len,
```

Comment:
If this is excessive, I don't mind removing it. I just wanted to prove equivalence to some "KAT" and the ones from https://datatracker.ietf.org/doc/draft-ietf-lamps-dilithium-certificates/ serve as exactly that -- except the standard doesn't provide DER/RAW representations. Since I'm building the entire KAT from just the seed, it seems a shame not to reach a full known answer. Plus, this tests out ASN.1 parsing of seed format, rather than full format which is done as part of the X.509 parsing tests (I think it's X.509).